content/docs/atom-guide/look-dev/materials/_index.md

```diff
@@ -11,6 +11,7 @@ For more technical details, see [Atom Developer Guide - Material System](/docs/a
 
 | Topic                        | Description |
 |--------------------------------------|---------|
+| [Overview](materials/) | An overview of the material system. |
```

Comment:
If you change the file name as suggested in file `content/docs/atom-guide/look-dev/materials/materials.md`, make sure to update this too.